mdata/cache/accnt/lru.go

```diff
@@ -0,0 +1,43 @@
+package cache
+
+import (
+	"container/list"
+)
+
+type entry struct {
```

Comment:
you're right, i can remove the `entry` struct and just directly pass the value around.